subsys/net/lib/aws_fota/src/aws_fota.c

```diff
@@ -111,115 +111,192 @@ static int update_job_execution(struct mqtt_client *const client,
 	return ret;
 }
 
-
-static int aws_fota_on_publish_evt(struct mqtt_client *const client,
-				   const u8_t *topic,
-				   u32_t topic_len,
-				   u32_t payload_len)
+/**
+ * @brief Parsing an AWS IoT Job Execution response recived on $next/get MQTT
+ *	  topic or notify-next if it is a valid response the program state is
```

Comment:
Add some puncutation before "if it is..."

Reply:
Fixed